src/etc/inc/plugins.inc.d/ipsec.inc

```diff
@@ -1786,6 +1691,7 @@ function ipsec_configure_vti($verbose = false, $device = null)
         }
         if (
             empty($configured_intf[$intf])
+            || empty($intf_details['tunnel'])
```

Comment:
what's the context here?

Reply:
the two lines below,  if `tunnel` doesn't exist, this `$local_configured != $intf_details['tunnel']['src_addr']` is always true, but will fail with a php error (at least as of php 8).

Reply:
👍